### PR TITLE
USSD registered user data storage

### PR DESCRIPTION
**Pull request by imsickofmaps**
- The "complete" flag should be added at initial_state with value "false"
- The "questions" array should be added at initial_state with value "[]"
- The registration date should be saved to contact
- The "complete" flag should be marked "true" on save

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@smn ready for review

File: js_sandbox/lib/go-nike-ghr.js [hunk 1]
```diff
@@ -20,16 +20,87 @@ function GoNikeGHR() {
     // The first state to enter
     StateCreator.call(self, 'initial_state');
 
+    self.get_today = function(im) {
+        var today = null;
```
Comment (smn) [inline js_sandbox/lib/go-nike-ghr.js]: unused.

File: js_sandbox/lib/go-nike-ghr.js [hunk 2]
```diff
@@ -56,14 +127,50 @@ function GoNikeGHR() {
...
-            return new ChoiceState(
-                state_name,
-                'end_state',
-                "Thank you for registering",
-                [
-                    new Choice("continue", "Continue")
-                ]
-            );
+            // Get the user
+            var p = im.api_request('contacts.get_or_create', {
```
Comment (smn) [inline js_sandbox/lib/go-nike-ghr.js]: if you're doing this a lot perhaps write a helper function?

**Comment by smn** [comment]
:+1: except minor issues

**Pull request merged**